platform/logging/LoggingConfig.ts

```diff
@@ -0,0 +1,82 @@
+import * as schema from '../lib/schema';
+import { typeOfSchema } from '../types';
+import { LogLevel } from './LogLevel';
+import { BaseAppenderConfig, BaseAppenderConfigSchemaType } from './appenders/base/BaseAppenderConfig';
+import { ConsoleAppenderConfig } from './appenders/console/ConsoleAppenderConfig';
+import { FileAppenderConfig } from './appenders/file/FileAppenderConfig';
+import { RollingFileAppenderConfig } from './appenders/rolling-file/RollingFileAppenderConfig';
+
+const KNOWN_APPENDERS = new Map<string, new (schema:  BaseAppenderConfigSchemaType) => BaseAppenderConfig>([
```

Comment:
What's the ES approach here? If ES doesn't, we likely shouldn't either. Most things can be solved using beats etc, where there is already a great system for handling these things